docs/AUTHORS.data.txt

```diff
@@ -148,6 +148,7 @@ Scenarios:
  kill_the_magic             by Titus Tscharntke (titi)
  magic_can_do_it            by Titus Tscharntke (titi)
  my_island                  by Titus Tscharntke (titi)
+ native_unrest              by Julie Marchant
```

Comment:
The AUTHORS.md in the source repo will be more prominent. You are welcome to add your name and link there as well.

https://github.com/ZetaGlest/zetaglest-source/blob/develop/AUTHORS.md